README.md

```diff
@@ -1,8 +1,8 @@
-This repo maintains the lists of repositories defining ROS distributions.
+This repo maintains the lists of repositories defining ROS 2 distributions.
 
-It is the implementation of [REP 143](http://ros.org/reps/rep-0143.html)
+It is an implementation of [REP 143](http://ros.org/reps/rep-0143.html)
 
-It also the home of the rosdep rules.
+The rosdep database in https://github.com/ros/rosdistro should still be used for ROS 2.
```

Comment:
`should still be used ` -> is still used? is also used?

Reply:
may also be worth pointing to the rosdep folder of rosdistro rather than the repo itself